devbox/src/syncer/Syncer.scala

```diff
@@ -15,8 +15,16 @@ class Syncer(agent: AgentApi,
              ignoreStrategy: String = "dotgit",
              debounceMillis: Int,
              proxyGit: Boolean,
-             signatureTransformer: (os.SubPath, Sig) => Sig)
+             signatureTransformer: (os.SubPath, Sig) => Sig,
+             syncIgnore: Option[String])
             (implicit ac: castor.Context, logger: SyncLogger) extends AutoCloseable{
+
+
+  val syncIgnoreRegex = syncIgnore.map(com.google.re2j.Pattern.compile(_))
+  pprint.log(syncIgnore)
+  val dummy = "apiproxy/deploy/multitenant/zzz_JSONNET_GENERATED/azure/prod/az-brazilsouth/mt-shard/apiproxy.yaml"
+  pprint.log(syncIgnoreRegex.map(_.matches(dummy)))
```

Comment:
Is this for debugging?

Reply:
I guess the above would make for a good test ;-)

Reply:
Haha yeah, I should kill this